ydb/src/grpc_wrapper/raw_auth_service/client.rs

```diff
@@ -4,23 +4,23 @@ use crate::grpc_wrapper::raw_auth_service::login::RawLoginResult;
 use crate::grpc_wrapper::raw_auth_service::login::RawLoginRequest;
 use crate::grpc_wrapper::runtime_interceptors::InterceptedChannel;
 use crate::grpc_wrapper::raw_errors::RawResult;
+use crate::grpc_wrapper::raw_services::{GrpcServiceForDiscovery, Service};
 
 use tracing::trace;
 
-#[allow(dead_code)]
+#[derive(Clone)]
 pub(crate) struct RawAuthClient {
     service: AuthServiceClient<InterceptedChannel>,
 }
 
 impl RawAuthClient {
-    #[allow(dead_code)]
     pub fn new(service: InterceptedChannel) -> Self {
         Self {
             service: AuthServiceClient::new(service),
         }
     }
 
-    #[allow(dead_code)]
+    #[tokio::main(flavor = "current_thread")]
```

Comment:
it is wrapper only, the code must be work with any tokion flavour.

Use `#[tokio::main(flavor = "current_thread")]` is good idea, but not very good place.

You should create runtime (use the attribute) on border between async and sync code. In that case - better create additional function in UserPasswordAuth struct as proxy (for create async runtime).